pyproject.toml

```diff
@@ -7,15 +7,17 @@ license = "MIT"
 
 [tool.poetry.dependencies]
 python = ">=3.9, <3.10"
-pandas = "^1.5.1"
 boto3 = "^1.26.0"
-boto3-stubs = {extras = ["s3"], version = "^1.25.5"}
+pandas = "^1.5.1"
+
 
 [tool.poetry.dev-dependencies]
+boto3-stubs = {extras = ["s3"], version = "^1.25.5"}
```

Comment:
❓ I think `boto3-stubs` needs to be a dependency rather than a dev-dependency, since `fetch_events.py` imports type definitions from the [`mypy_boto3_s3` package](https://youtype.github.io/boto3_stubs_docs/mypy_boto3_s3/), which is installed via `boto3-stubs`. I ended up doing all this because `boto3` doesn't come with type definitions. Let me know if I'm way off, though!